Makefile

```diff
@@ -62,23 +68,24 @@ O6-psds = \
 .SECONDEXPANSION:
 psd_files = $(sort $(filter-out --%,$(value $(1)-psds)))
 runs/%/psds.xml: $$(call psd_files,%)
-	mkdir -p $(@D) && scripts/pack-psds.py -o $@ $(value $*-psds)
+	mkdir -p $(@D) && scripts/pack-psds.py -o $@ --config $* $(value $*-psds)
 
 
 #
 # Download samples from the Farah distribution.
 # FIXME: this document must be public!
 #
 
-O1O2O3all_mass_h_iid_mag_iid_tilt_powerlaw_redshift_maxP_events_all.h5:
-	curl -OL https://dcc.ligo.org/LIGO-T2100512/public/O1O2O3all_mass_h_iid_mag_iid_tilt_powerlaw_redshift_maxP_events_all.h5
+
+GWTC4_BrokenPowerLawTwoPeaks_baseline5_population.hdf5:
+	curl -OL https://dcc.ligo.org/public/0203/T2500311/004/GWTC4_BrokenPowerLawTwoPeaks_baseline5_population.hdf5
 
 
 #
 # Convert the Farah samples to the format needed by bayestar-inject.
 #
 
-farah.h5: O1O2O3all_mass_h_iid_mag_iid_tilt_powerlaw_redshift_maxP_events_all.h5 scripts/farah.py
+farah.h5: GWTC4_BrokenPowerLawTwoPeaks_baseline5_population.hdf5 scripts/farah.py
```

Comment:
Please rename this target, as it is no longer accurate to describe this as the "Farah" distribution.